modules/oe_theme_helper/src/ContentUtility.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_helper;
+
+use Drupal\Core\Field\EntityReferenceFieldItemListInterface;
+
+/**
+ * Utility class for content data.
```

Comment:
```suggestion
 * Provides field value-related conversions and operations.
```
I'm shying away from "content data" because below we are clearly handling fields and their values.